tests/unit/widget-filter-simple-condition.spec.ts

```diff
@@ -55,6 +64,16 @@ describe('Widget WidgetAggregation', () => {
     expect(widgetWrappers.at(1).props().value).to.equal('be among');
   });
 
+  it('should channge the type of value accordingly when switching the "operator"', async () => {
+    const wrapper = shallowMount(WidgetFilterSimpleCondition, { store: emptyStore, localVue });
+    expect((wrapper.vm.$data.editedValue.value = ''));
+    wrapper.setData({ editedValue: { column: 'foo', value: '', operator: 'in' } });
+    await localVue.nextTick();
```

Comment:
same here

Reply:
This one works fine without `nextTick`...